s3tbx-sentinel3-reader/src/main/java/org/esa/s3tbx/dataio/s3/meris/MerisLevel1ProductReader.java

```diff
@@ -17,7 +17,7 @@ public MerisLevel1ProductReader(MerisLevel1ProductPlugIn readerPlugIn) {
     @Override
     protected Product readProductNodesImpl() throws IOException {
         final String dirName = getInputFileParentDirectory().getName();
-        if (dirName.matches("EN.*_(F|R)R(G|P|S).*")) {
+        if (dirName.matches("EN(1|V).*_ME(R|_)1?_(F|R)R(G|P|S|_)_(1P)?_.*")) {
```

Comment:
I think you have extended the expression to match the zip file names like 
EN1_MDSI_MER_RR__1P_20111230T102706_20111230T111042_051430_0123_20180901T181455_0100.ZIP
But actually, this is not the folder name according to the format.
https://earth.esa.int/eogateway/documents/20142/37627/MERIS-Sentinel-3-Like-L1-andL2-PFS.pdf (section 3.2 page 11).

However, probably it does not disturb the usual reading. But it might be good to split both cases and make the expressions more specific to make it clearer. Also a unit level test would be good.